### ISSUE TITLE
New editorConfig option doesn't seem to work or maybe missing themes?

### ISSUE THREAD

**Issue by bluetidepro** [styleguidist/react-styleguidist#788]
In the new minor update release (`v6.2.0`) notes it mentions moving from `highlightTheme` to use the `editorConfig` option, but it doesn't seem to work. Or maybe themes were removed?

I used to have `highlightTheme: 'material',`, and then I switched it to: 

```
  editorConfig: {
    theme: 'material',
  },
```

It would look like http://bluetide.pro/FKiiDg when I used the `highlightTheme` option, but now when I use the `editorConfig` it looks like http://bluetide.pro/SBR33b

Have themes been removed apart of this update?

**Comment by vuongc** [comment]
I've got the same error, it only work for the default theme which is `base16-light`.

**Comment by bluetidepro** [comment]
It's worth noting that if I still use `highlightTheme: 'material',` it DOES work fine. Although it gives me this warning: `Warning: highlightTheme config option is deprecated. Use the theme property in the editorConfig option instead`. So maybe it DOES has to do with the new `editorConfig` option?

**Comment by sapegin** [comment]
Yeah, the old options is still used in couple of places:

https://github.com/styleguidist/react-styleguidist/blob/11bbe2cafab723efb159833fe831ccfb1e5499ec/scripts/make-webpack-config.js#L34

https://github.com/styleguidist/react-styleguidist/blob/5a5a3762a78873120bd4bbc62272a7bd36e67681/loaders/styleguide-loader.js#L19

/cc @SaraVieira

**Comment by davidjb** [comment]
@sapegin Fixed in PR #814 -- could you take a look?  The file `make-webpack-config.js` seems to be the only spot where `highlightTheme` was used over the `editorConfig` option.  The PR is backwards compatible; it uses `editorConfig.theme` if present, and falls back otherwise.
